nixos/tests/environment-skel.nix

```diff
@@ -0,0 +1,29 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "environment-skel";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ mkg20001 ];
+  };
+
+  nodes = {
+    machine = { config, lib, pkgs, ... }: {
+      users.users.testuser = {
```

Comment:
shouldn't copySkel be set?

Reply:
It's implict if createHome is set, but I think it should be implicitly set if createHome && isNormalUser

This tries to replicate the behaviour of the adduser command which automatically copies skel unless otherwise specified AFAIK